Composer/packages/test-utils/index.js

```diff
@@ -3,10 +3,12 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 
 const testingLib = require('@testing-library/react');
+const hooksLib = require('@testing-library/react-hooks');
 
 const createConfig = require('./createConfig');
 
 module.exports = testingLib;
+module.exports.hooks = hooksLib;
```

Comment:
I would like to have done something like 

```ts
import { act } from '@bfc/test-utils/hooks'`
```

But I couldn't figure out how to expose that in the typings correctly. If anyone has some TS wizardry, I would appreciate.